codecov.yml

```diff
@@ -1,26 +1,20 @@
 coverage:
   status:
-    patch:
-      default:
-        target: 0%
-        threshold: 0%
-      core-packages:
-        target: 0%
-        paths:
-          - packages
-      plugins:
-        target: 0%
-        paths:
-          - plugins
+    patch: off
     project:
       default:
         target: 0%
-        # send notification if coverage change exceed this threshold
-        threshold: 0%
-      core-packages:
+        threshold: 10%
+      core-packages-ts:
         target: 100%
         paths:
-          - packages
+          - packages/*.ts$
```

Comment:
> The pattern folder/* will not match recursively in the folder.
> Please use this folder/**/*.

https://docs.codecov.io/docs/ignoring-paths

Reply:
yeah, I was referring to https://docs.codecov.io/docs/path-regexp and thought it’d work.